conf/modules/parabricks_fq2bammeth.config

```diff
@@ -0,0 +1,12 @@
+process {
+    withName: PARABRICKS_FQ2BAMMETH {
+        cache = 'lenient' // This is set because in the module command the index files are touched so as to have bwameth not complain
+        ext.args = '--low-memory'
```

Comment:
`--low-memory` might not be appropriate for production.

Reply:
I think its a limitation of parabricks for now. It needed that for when I was testing with a prod run too

Reply:
Gotcha. It might be related to the EC2 instance being used. Let's pass it for now.